docs/develop/go/debugging.mdx

```diff
@@ -67,270 +71,3 @@ For more information, see [Metrics](/develop/go/observability#metrics) for setti
 
 Debug Server performance with [Cloud metrics](/cloud/metrics/) or [self-hosted Server metrics](/self-hosted-guide/production-checklist#scaling-and-metrics).
 
-## How to test Workflow Definitions in Go {#testing-and-debugging}
```

Comment:
This is a HUGE HUGE removal of content. Was that intentional?

Reply:
Putting this here: Any removals are likely because they are being moved to a new spot to align with the migration and IA rework of the dev guides. The .NET one was a manual migration since the rest weren't, we need to make sure they are still manually moved. ✔️